src/vocab/services/extension-vocab.js

```diff
@@ -54,15 +54,17 @@ class ExtensionVocab extends ReplaySubject {
   }
 
   /**
-   * Restore words from the storage
+   * Restore words from local storage
+   *
+   * @see storageKey
    */
   restoreSavedWords() {
     const savedWords = localStorage.getItem(storageKey);
     savedWords && this.setWords(JSON.parse(savedWords));
   }
 
   /**
-   * Update words and save into the storage
+   * Updates the words and saves them into local storage.
```

Comment:
I would keep the verbs in the imperative form as is the convention in many codebases/languages.